### ISSUE TITLE
How to write into a pointer inside a callback?

### ISSUE THREAD

**Issue by neko-para** [Koromix/koffi#77]
I need to pass a callback that would write into the pointer param.
The callback has the prototype `int8_t (*get_resolution)(int32_t* width, int32_t* height)`.
I know that I can use decode to get the value, but I cannot **set** it.
```javascript
const width = koffi.decode(ptrwidth, koffi.array('int32_t', 1, 'Array'))
width[0] = 111 // this cannot and shouldn't write into *width, at least.
```

**Comment by neko-para** [comment]
After all, memcpy does help dealing this situation.
```javascript
memcpy(ptrwidth, [111], 4)
```
But I still wonder if there would be a better solution

**Comment by Koromix** [comment]
I will look into implementing something like `koffi.encode()` soon.
The memcpy is a good solution in the mean time :)

**Comment by Koromix** [comment]
Can you try `Koffi 2.5.21-beta.2`?

```js
koffi.encode(ptrwidth, 'int32_t', 123);
koffi.encode(ptrheight, 'int32_t', 321);
```

**Comment by Koromix** [comment]
I've published Koffi 2.6.0 with all of this.

Documentation: https://koffi.dev/variables
Changelog: https://koffi.dev/changelog
